lib/posix/timer.c

```diff
@@ -16,7 +17,12 @@ static void zephyr_timer_wrapper(struct k_timer *ztimer);
 
 struct timer_obj {
 	struct k_timer ztimer;
+	int sigev_notify;
 	void (*sigev_notify_function)(sigval val);
+	struct k_sem sem_cond;
+	pthread_t thread;
+	pthread_attr_t attr;
```

Comment:
This is unecessary to keep, and can be discarded after `pthread_create()` is called